### DISCUSSION TITLE
Send Later issues while using Aluco addon (Outlook Web Access - OWA)

### DISCUSSION THREAD

**Discussion by khunalex** [Extended-Thunder/send-later#530] [Q&A] [answered]
Hi.

I am using OWL - Aluco addon for Thunderbird to be able to read and send emails with Microsoft365. 
When I schedule an email witth "Send Later", always get prompted for an administrator autorization and fails.
Could it be that  "Send Later" its using a different method to send the emails skipping the OWA protocol?

Thank you for your feedback.

**Comment by khunalex** [comment]
Send Later is not working with OWA (Office Web Access in Thunderbird,
because Send Later forces to send the email via default SMTP. However
OWL/Aluco doesnt use SMTP, since it works OWA.
Thats why mails can be sent manually but they dont go through if scheduled
with Send Later.

**Comment by Bluestone357** [comment]
I too struggle with Thunderbird +/- SL and 365.
Using Thunderbird I can compose, amend drafts, delete, but **NOT** send.
Not send with or without SL.

**Comment by khunalex** [comment]
To Send should be fine. Just delete the SMTP because OWL doesn't use it,
since it's OWA.
I had the same issue, and to delete the SMTP worked for me. However Send
Later try to send all the emails via default smtp. That's why doesn't work.

**Comment by jikamens** [comment, marked as answer]
As far as I know the only way to get Send Later to work with Owl is via SMTP. This is [documented in the Send Later user guide](https://extended-thunder.github.io/send-later/#owl-for-exchange-add-on). The behavior you are describing here is as designed.
